<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelDatasCharacter</class>
 <widget class="QWidget" name="PanelDatasCharacter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,1">
       <item>
        <widget class="QLabel" name="labelClass">
         <property name="text">
          <string>Class:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxClass">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxFaceset">
         <property name="title">
          <string>Faceset</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0" colspan="3">
             <widget class="WidgetPicture" name="widgetPictureFaceset" native="true">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>98</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <widget class="QScrollArea" name="scrollArea">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>98</width>
                 <height>98</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="WidgetShowPicture" name="widgetImageFaceset" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item row="1" column="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>28</width>
                <height>98</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxBattler">
         <property name="title">
          <string>Battler</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="WidgetPicture" name="widgetPictureBattler" native="true">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QScrollArea" name="scrollArea_2">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_2">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>337</width>
                 <height>92</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="WidgetShowPicture" name="widgetImageBattler" native="true"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="PanelDatasClass" name="panelDatasClass" native="true">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetPicture</class>
   <extends>QWidget</extends>
   <header>widgetpicture.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanelDatasClass</class>
   <extends>QWidget</extends>
   <header>paneldatasclass.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetShowPicture</class>
   <extends>QWidget</extends>
   <header>widgetshowpicture.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
